skunkworks_crow/src/main/java/org/odk/share/activities/ReceiverActivity.java

```diff
@@ -491,12 +495,12 @@ private void wifiListAvailable(ArrayList<WifiNetworkInfo> list) {
         if (isQRCodeScanned) {
             isQRCodeScanned = false;
             for (WifiNetworkInfo info : list) {
-                if (info.getSsid().equals(ssidScanned)) {
+                if (info.getSsid().equals(wifiNetworkSSID)) {
                     if (info.getState() != NetworkInfo.DetailedState.CONNECTED) {
                         Toast.makeText(this, "attempting connection", Toast.LENGTH_SHORT).show();
-                        connectToNetwork(info.getSecurityType(), ssidScanned, passwordScanned);
+                        connectToNetwork(info.getSecurityType(), wifiNetworkSSID, passwordScanned);
                     } else {
-                        Toast.makeText(this, "already connected to " + ssidScanned, Toast.LENGTH_SHORT).show();
```

Comment:
Can we extract the toasts into something like `ToastUtils.java`? You can refer to [Collect's](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/java/org/odk/collect/android/utilities/ToastUtils.java). Something for showing dialogs can be refactor as well.